.eslintrc.js

```diff
@@ -0,0 +1,178 @@
+const eslintConfig = {
+	env: {
+		es6: true,
+		node: true,
+	},
+	plugins: ['prettier'],
+	settings: {},
+	parserOptions: {
+		ecmaVersion: 2022,
+		sourceType: 'module',
+		ecmaFeatures: {
+			impliedStrict: true,
+		},
+		lib: ['es2022'],
+	},
+	globals: {
+		NodeJS: 'readonly',
+	},
+	extends: ['eslint:recommended', 'plugin:prettier/recommended'],
+	rules: {
```

Comment:
I have not worked with these for some time, just wondering if it is necessary to have so many manual `no-` things in there, maybe there is a preset for this? Also not clear what has changed in the config in the way things are committed, would have been great to see one commit where file is renamed, and separate when it is changed. "change everything" approach is very hard to review.

Reply:
All the `no-xxx` options here were carefully selected in the past to satisfy our wishes and fears. Honestly I don't want to use a "preset" and adhere to it. Anyway, I'll explain in a separate comment the rationale behind changes done here.